test/test_network_constructors.jl

```diff
@@ -166,7 +166,12 @@ end
     c_sys14_dc = build_system("c_sys14_dc")
     systems = [c_sys5, c_sys14, c_sys14_dc]
     objfuncs = [GAEVF, GQEVF, GQEVF]
-    constraint_names = [:RateLimitFT__Line, :RateLimitTF__Line]
+    constraint_names = [
+        :RateLimitFT__Line,
+        :RateLimitTF__Line,
+        :Bus_active_power_balance,
+        :Bus_reactive_power_balance,
```

Comment:
Why is free-form text being used here instead of the const strings defined in src/core/definitions.jl?  This practice will make it very hard to debug problems because we won't be able to find where specific items are used.

Reply:
@claytonpbarrows is there a reason for this?

Reply:
Also, constraint names should use the format `name__DeviceType`

Reply:
agreed.

Reply:
I think this was never updated from the original PM interface. I'll try to address.

Reply:
You might as well call the function `make_constraint_name`, which is new in `dev0-6`.

Reply:
I'll create another PR to address that